### ISSUE TITLE
Should `scoped` support be dropped?

### ISSUE THREAD

**Issue by jods4** [aurelia/templating-resources#270]
`aurelia-pal` has `FEATURE.scopedCSS` and `aurelia-templating-resources` tries to add `scoped` to injected `<style>` when appropriate (e.g. `<require from="control.css">`).

I wonder if this should be removed from Aurelia, to re-align with the current HTML specs.
`scoped` was dropped from the specs (in favor of shadow DOM) and the only browser that implements it (at least for now) is Firefox.

**Comment by thomas-darling** [comment]
I agree that something that is no longer on a standards track and only works in one browser anyway, should probably be removed. However, that leads to another question:  

What is the recommended way to achieve CSS scoping, and what are the thoughts on it going forward?
I can't seem to find anything in the docs related to this.

**Comment by jods4** [comment]
At the time I opened this ticket, `scoped` was supported by Firefox, meaning it would have been a breaking change to remove it, as someone could have built an Aurelia app targetting FF only.

Since then, FF has moved that feature behind a config flag in FF 55, which means it works in _no browser at all_ today.
This is dead code and I think at this point we should remove it.

@thomas-darling Standard that have replaced scoped styles is shadow DOM. Styles inside a shadow DOM don't bleed out (and vice-versa). You can read more about it here, go to _Styling_ section:
https://developers.google.com/web/fundamentals/web-components/shadowdom

Non-standard stuff that I find interesting is _CSS Modules_, google it.

**Comment by Alexander-Taran** [comment]
if the decision is to drop it, help_needed and prs_welcome are appropriate tags for this.

**Comment by Alexander-Taran** [comment]
Ok there is a silent consensus that this should be removed.
As nobody objects.
Is is a ctrl-a shift-del operations?
Or it needs a week to carefully delete some lines of code?

**Comment by Alexander-Taran** [comment]
<img width="1374" alt="image" src="https://user-images.githubusercontent.com/1029452/171293823-9f755cfc-a5eb-4cd1-ae72-676c8c615bb9.png">

@bigopon
